controller/aks-cluster-config-handler.go

```diff
@@ -308,13 +318,11 @@ func (h *Handler) checkAndUpdate(config *aksv1.AKSClusterConfig) (*aksv1.AKSClus
 	for _, np := range *result.AgentPoolProfiles {
 		if status := to.String(np.ProvisioningState); status == NodePoolCreating ||
 			status == NodePoolScaling || status == NodePoolDeleting || status == NodePoolUpgrading {
-			if config.Status.Phase != aksConfigUpdatingPhase {
-				config = config.DeepCopy()
-				config.Status.Phase = aksConfigUpdatingPhase
-				config, err = h.aksCC.UpdateStatus(config)
```

Comment:
There's nothing anymore that sets Phase to Updating if only node pools are changing, is that intended?

Reply:
I believe this will change the status when a node group is updated: https://github.com/thedadams/aks-operator/blob/cf75e6d76643265b31f4c791edd5959c1fb1190b/controller/aks-cluster-config-handler.go#L778